### DISCUSSION TITLE
Install question: setting Qt version.

### DISCUSSION THREAD

**Discussion by SirElismyer** [textbrowser/biblioteq#90] [Q&A] [answered]
Clearly I am an idiot. I've been trying for the last bit to get BiblioteQ up and running but so far have been unsuccessful. Running the Bash script in the shell throws this flag:
`./BiblioteQ: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version 'Qt_5.12' not found (required by ./BiblioteQ)`

I've installed QT 5.12 as part of the provided administrative instructions but the libQT5Core.so.5 is pointing toward 5.9.5. I have no idea how to change this and Synaptic is not giving me any help. 

Running Linux Mint 19.1.

**Comment by textbrowser** [comment]
OK, so the binary provided with the Debian file is created using Qt 5.12.

In the script, set LD_LIBRARY_PATH to the location of your Qt 5.12 libraries. For example: export LD_LIBRARY_PATH=/path/to/Qt/lib.

**Comment by textbrowser** [comment]
Correction, the Dooble release includes Qt 5.15. Anyway, how did you get Dooble?

**Reply by textbrowser** [reply]
Oops, I was thinking of another application.

So yah, does your Mint version include Qt 5.9 and you installed Qt 5.12 using Qt? If so, LD_LIBRARY_PATH is a solution.

**Comment by textbrowser** [comment, marked as answer]
You can also practice building BQ from source.
